### DISCUSSION TITLE
How to output yaml format data?

### DISCUSSION THREAD

**Discussion by 180909** [apiflask/apiflask#21] [Q&A] [answered]
Hello, Greyli, thanks for you job in Python web framework. I visit the apiflask.com and i find the function `get_spec(self, spec_format='json')`.But i can't use it in my code. Can you show me how to use this function. Thank you very much.

**Comment by greyli** [comment, marked as answer]
This function is meant for internal use.

To get the spec, you can visit `/openapi.json` (i.e. http://localhost:5000/openapi.json). When you change the extension of the  `spec_path` argument to `.yml` or `.yaml`, the YAML format will be returned:

```
app = APIFlask(__name__, spec_path='/openapi.yaml')
``` 

Now, visit http://localhost:5000/openapi.yaml.

**Reply by 180909** [reply]
Thank you for your answer.
